typescript.md

```diff
@@ -98,7 +104,7 @@ function printLabel(options: LabelOptions) { ... }
 ```ts
 interface User {
   name: string,
-  age?: number
+  age?: number // can be a number or undefined
```

Comment:
Hmm, this isn't `number or undefined`, but rather it's possible that `age` would be absent. `age: undefined` would be an error :)

Reply:
Right. I updated it.